docs/_ug/KeyDefinitions.md

```diff
@@ -14,7 +14,7 @@ The following are the attributes stored for each item:
 * Item remarks
 * Item tags
 
-FoodRem Items are unique by name and case-sensitive. This means you cannot add two or more items of the same name.
+FoodRem Items are unique by name and case-insensitive. This means you cannot add two or more items of the same name.
```

Comment:
They are case sensitive. "Potato" and "potato" are different. Right? @Eugenetanwl3881

Reply:
Right now, yes